helper/dhutil/dhutil.go

```diff
@@ -90,7 +90,6 @@ func DeriveSharedKey(secret, ourPublic, theirPublic []byte) ([]byte, error) {
 	if n != 32 {
 		return nil, errors.New("short read from hkdf")
 	}
-	fmt.Printf("Key: %s\n", hex.EncodeToString(key[:]))
```

Comment:
Looks like this was the only thing using "encoding/hex" so the import can be removed?

Reply:
Ah, thanks. That's what I get for editing in a different editor than usual. :)